tests/main.test.ts

```diff
@@ -95,53 +95,52 @@ test('test action run diff correctly', async () => {
     process.env.INPUT_FILE,
   );
 
-  expect(mockedDiff.run).toHaveBeenCalledWith([
+  expect(mockedDiff.prototype.run).toHaveBeenCalledWith(
     'my-file-to-diff.yml',
-    '--doc',
+    undefined,
```

Comment:
This is possible since neither `doc` neither `token` are required anymore in `action.yml`,  right ?

Reply:
No, it's not related.

This is the main change of the PR: instead of using a “command” object from the bump-cli, we use the new dedicated “CoreDiff” library. Thus the signature of the `run` function changes:
- With the “command” object, we needed to pass an array of command parameters: `run([file, "--doc", doc_name, "--token", token])` 
- With the “CoreDiff” library, we need to pass parameters to the `CoreDiff.run` function. Cf the function signature:
  ```ts
  public async run(
    file1: string,
    file2: string | undefined,
    documentation: string,
    hub: string | undefined,
    token: string,
  ): Promise<VersionResponse | undefined> {
  ```
  thus, the parameters are now: `run(file, undefined, doc_name, undefined, token)`

Reply:
Indeed, thanks for your explanation